packages/opal-common/opal_common/git/tests/bundle_maker_test.py

```diff
@@ -267,3 +267,49 @@ def test_bundle_maker_sorts_according_to_explicit_manifest(local_repo: Repo, hel
     assert "other/abac.rego" == bundle.manifest[1]
     assert "rbac.rego" in bundle.manifest
     assert "other/data.json" in bundle.manifest
+
+def test_bundle_maker_sorts_according_to_explicit_manifest_nested(local_repo: Repo, helpers):
+    """
+    Test bundle maker filtered on directory only returns opa files from that directory
+    """
+    repo: Repo = local_repo
+    root = Path(repo.working_tree_dir)
+
+    # Create multiple recursive .manifest files
```

Comment:
This test is great, but we also need some tests for failure cases. What happens when a manifest points to another manifest that doesn't exist? What if the manifest is binary instead of utf-8 text? What if the manifest is a symbolic link? What if we have a manifest cycle?